content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -255,9 +255,6 @@ different Kubernetes components.
 | `CSIServiceAccountToken` | `false` | Alpha | 1.20 | 1.20 |
 | `CSIServiceAccountToken` | `true` | Beta | 1.21 | 1.21 |
 | `CSIServiceAccountToken` | `true` | GA | 1.22 | |
-| `CronJobControllerV2` | `false` | Alpha | 1.20 | 1.20 |
-| `CronJobControllerV2` | `true` | Beta | 1.21 | 1.21 |
-| `CronJobControllerV2` | `true` | GA | 1.22 | - |
```

Comment:
I don't think we should remove this.

Reply:
@tengqm I did wonder about that too, but on the other hand that value is not present anymore, so you can't set it. It'll make senese for previous versions of k8s, but those have their own docs, that's why I've decided to go with it.

Reply:
I agree with @tengqm , these are left in even though we maintain docs for the current + past 4 releases

Reply:
Fair, I'm closing the PR in that case.